docs/HowToFAQ/FAQNonlinearArith.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Are there any useful tricks (e.g. z3 switches) to get better automation for nonlinear arithmetic (perhaps trading off other performance aspects)?
+---
+
+## Question
+
+Are there any useful tricks (e.g. z3 switches) to get better automation for nonlinear arithmetic (perhaps trading off other performance aspects)?
+
+## Answer
+
+No. There are some switches (/arith and the somewhat deprecated /noNLarith) that reduce the automation you get with nonlinear arithmetic. There are now also many lemmas about nonlinear arithmetic in the Dafny standard library.
```

Comment:
I think we could rephrase this into a more general question: "I'm having issues with non-linear arithmetic.  How can I improve things?"

Then the suggestion is indeed to use `/arith` or `/noNLarith` + the stdlib